nearform/mercurius-federation-info-graphiql-plugin
# feat: Update table view layout according to design

This PR closes #82 and #115

The PR introduces the material UI components in the "Overall schema" panel and:
- adds column sorting to all the tables.
- removes the `SASS` package from the project because all the `scss` files have been removed.
- saves the open accordions and subtables in the local storage (using the `PluginState` context).

![Screenshot 2022-12-15 at 19 07 49](https://user-images.githubusercontent.com/1233582/207935416-ead99530-523a-4125-87d9-a7d78bc9631a.png)


## How to test it

To test this PR:
- install the dependencies with `npm ci` 
- execute the command `npm run example` 
- open your browser to http://localhost:3001/graphiql
- then click on the plugin tab

Please note that this solution does not require the `mercurius-federation-info` project to run because the plugin example builds the source code and runs a mercurius instance for you.
